analyzers/src/SonarAnalyzer.Common/Helpers/KnownType.cs

```diff
@@ -212,6 +212,7 @@ public sealed partial class KnownType
         public static readonly KnownType NUnit_Framework_TestCaseSourceAttribute = new("NUnit.Framework.TestCaseSourceAttribute");
         public static readonly KnownType NUnit_Framework_TestFixtureAttribute = new("NUnit.Framework.TestFixtureAttribute");
         public static readonly KnownType NUnit_Framework_TheoryAttribute = new("NUnit.Framework.TheoryAttribute");
+        public static readonly KnownType Newtonsoft_Json_JsonPropertyAttribute = new("Newtonsoft.Json.JsonPropertyAttribute");
```

Comment:
This should be before the NUnit types (alphabetically). The same for the package reference.